forge-game/src/main/java/forge/game/spellability/LandAbility.java

```diff
@@ -37,11 +37,6 @@ public LandAbility(Card sourceCard) {
         getRestrictions().setZone(ZoneType.Hand);
     }
 
-    public boolean canPlay(Card newHost) {
```

Comment:
@tool4ever why did this got removed?

Reply:
unused function that's just a pass-through